### ISSUE TITLE
Trouble getting it working

### ISSUE THREAD

**Issue by Sparticuz** [speechmarkdown/speechmarkdown-vscode#5]
I've installed the plugin in vscode, but am having trouble getting it working. I've created a .md file, and set the formatting to Speech Markdown. I've entered some text from an example and tried to convert it to SSML, but it's coming up blank. Am I doing something wrong?

![image](https://user-images.githubusercontent.com/161913/87687310-95f07c00-c753-11ea-916d-4f4edde2de4d.png)

**Comment by johniwasz** [comment]
Highlight the text and you'd like to see composed down to SSML.

![7EF222F5-3BB2-413C-9F28-4DEAAD321816](https://user-images.githubusercontent.com/44126651/87794568-d06e1d80-c814-11ea-91a1-17c72650cfec.GIF)

The extension is designed to be primarily used with JavaScript/TypeScript. Speech Markdown is supported by (Jovo)[https://www.jovo.tech/] and there is a (plug-in)[https://github.com/speechmarkdown/jovo-plugin-speechmarkdown] for Speech Markdown available.

![intellisense01](https://user-images.githubusercontent.com/44126651/87795600-4de65d80-c816-11ea-9c96-4cbf81119435.gif)

Do you have a use case where you would like to see support for Speech Markdown syntax highlighting in Markdown rather than JavaScript/TypeScript? 

**NOTE**: The [speechmarkdown-js](https://github.com/speechmarkdown/speechmarkdown-js) library does have some issues with parsing some syntax. Including "(quick)[rate:'x-fast'])" in the selection current results in:

```
Alexa: 
Error

Google Assistant: 
Error

Samsung Bixby: 
Error

Plain Text: 
Error
```

**Comment by Sparticuz** [comment]
My use case is probably easier than js/ts. I just want a file where I can type a text document (with speech markdown), and easily convert it to ssml. It's basically a line by line script for a training video.

![image](https://user-images.githubusercontent.com/161913/87803973-395b9280-c821-11ea-895e-371955e91655.png)

I've just typed this up in vscode, then ran the extension, but still not working. Including speech markdown, actually doesn't even work
![image](https://user-images.githubusercontent.com/161913/87804326-aec76300-c821-11ea-8e7f-b7ed55b8b9df.png)

**Comment by rmtuckerphx** [comment]
@Sparticuz Thanks for the feedback. 
Another place for you to check the syntax of Speech Markdown is: https://www.speechmarkdown.org/try/

Speech Markdown uses the word **number** instead of **cardinal** for non-ordinal numbers.

If you would like Speech Markdown to have cardinal be a synonym for number, then open an issue at:
https://github.com/speechmarkdown/speechmarkdown-js/issues

Thank you,

Mark

**Comment by Sparticuz** [comment]
Ah, I thought it was supported (https://www.speechmarkdown.org/syntax/cardinal/)

Either way, I know about the converter online, but it's not very useful for my use case.

**Comment by johniwasz** [comment]
@rmtuckerphx I am seeing a subtle issue.

This is processed successfully:

```
 (quick)[rate:"fast"]
```

This is not:
```
(quick)[rate:'fast'] 
```

The difference is double quotes vs single quotes.

Single quotes are in use when embedding SpeechMarkdown in JavaScript/TypeScript in some tags, like:

``` JavaScript
 MyNameIsIntent() {
    this.tell("#[voice:'Kendra';lang:'en-US'] Hello, nice to meet ++you++!");
  }
```

If I use double quotes in the sample above around Kenda and en-US it also processes successfully.

In both cases, the Alexa formatter  produces:

``` XML
<voice name="Kendra">
<lang xml:lang="en-US"> Hello, nice to meet <emphasis level="strong">you</emphasis>!</lang>
</voice>
```

So, it appears that some tags support double quotes only while others support both single and double.

@Sparticuz 

I'm in process of adding configuration settings so the opening and closing speaker tags are output. The setting will default to true and produce the speaker tags. I expect to have this update out over the weekend.

![image](https://user-images.githubusercontent.com/44126651/87841664-0642ee80-c875-11ea-9318-d3c3d607415d.png)

**Comment by johniwasz** [comment]
@Sparticuz 
Configuration changes have been applied to the latest release of the SpeechMarkdown VSCode extension (v0.0.9). Open and closing speech tags will now render in the output by default. The Include Speech Tags configuration option can disable that behavior if preferred.

**Comment by johniwasz** [comment]
With the update to reference speechmarkdown-js 1.8, this issue has been addressed in the 0.0.0.10 release of the Speech Markdown VS Code extension.
